tests/snippets/builtin_bin.py

```diff
@@ -0,0 +1,7 @@
+assert bin(0) == '0b0'
+assert bin(1) == '0b1'
+assert bin(-1) == '-0b1'
+assert bin(2**65) == '0b1' + '0' * 65
```

Comment:
At the moment we use 32 bit integers. There is an open point to change this to multiprecision arithmatic (for example https://gmplib.org/). For now, mark this test case as TODO by commenting and placing a TODO comment for it. We can only use 2**31 and less for now.